src/libAtomVM/opcodesswitch.h

```diff
@@ -7388,6 +7429,10 @@ HOT_FUNC int scheduler_entry_point(GlobalContext *glb)
                             DECODE_LITERAL(pattern_value, pc);
                             j++;
                             #ifdef IMPL_EXECUTE_LOOP
+                                if (size > 64) {
+                                    // TODO: implement support for big integers also here
```

Comment:
We probably should (a) have a test for this and (b) document it.

Reply:
Plan is still to fix this quite ASAP, so I'm not sure it is worth to be documented.